sites/fast-website/src/app/components/side-navigation/side-navigation.ts

```diff
@@ -20,21 +20,50 @@ export class SideNavigation extends FASTElement {
         x => x.header !== "Github"
     );
 
+    public scrollFunc = (num1: number, num2: number): void => window.scrollTo(num1, num2);
+
     public clickHandler = (e: Event, parent: boolean): void => {
         const target: HTMLElement = e.target as HTMLElement;
 
         const link: string = parent
             ? (target.getAttribute("href") as string)
             : (target.parentElement?.getAttribute("href") as string);
 
-        const selectedSection: Element = document.querySelector(link) as Element;
+        const selectedSection: HTMLElement = document.querySelector(link) as HTMLElement;
 
         e.preventDefault();
 
-        if (selectedSection) {
+        if (CSS.supports("scroll-behavior: smooth")) {
```

Comment:
This can be checked once and stored, rather than checking on every click.

Reply:
That is a great idea. Thank you. I have stored it in a variable.